src/c_api.rs

```diff
@@ -465,6 +465,33 @@ pub unsafe extern "C" fn c2pa_reader_resource_to_stream(
     }
 }
 
+/// Writes all the resources in a Reader to a folder.
```

Comment:
```suggestion
/// Writes all the resources from a Reader to a folder.
```
Your non-native reviewer got confused by the wording, but maybe my grammer is off.